decidim-meetings/app/views/decidim/meetings/meetings/_meeting.html.erb

```diff
@@ -0,0 +1,144 @@
+<%= render layout: "layouts/decidim/shared/drawer", locals: { close_url: meetings_path } do %>
```

Comment:
Accessibility violation on the meeting page:
![Meeting page a11y](https://user-images.githubusercontent.com/864340/209191558-5aa05a7a-c6c3-4996-8638-dc85000e869c.png)

![Meeting page a11y second](https://user-images.githubusercontent.com/864340/209193155-c7c070e8-9ddd-487b-843b-a7f262677e78.png)